_includes/testimonials.html

```diff
@@ -4,12 +4,20 @@ <h2 class="section-heading text-center">Testimonials</h2>
         <h3 class="section-subheading text-muted text-center">For confidentiality, some details are omitted</h3>
     </div>
 
+    <div id="testimonials-carousel" class="carousel slide" data-ride="carousel">
+        <!-- Indicators -->
+        <ol class="carousel-indicators">
+            <li data-target="#testimonials-carousel" data-slide-to="0" class="active"></li>
+            <li data-target="#testimonials-carousel" data-slide-to="1"></li>
+            <li data-target="#testimonials-carousel" data-slide-to="2"></li>
+        </ol>
+
     <div class="testimonials-carousel">
         <div>
             <div class="testimonial">
                 <h4>Testimonial</h4>
```

Comment:
I think we can basically move some of the stuff to matter and use the same for loop here?

Reply:
Agreed :) I'll do it in the next PR